src/Build/Version.txt

```diff
@@ -1 +1 @@
-1.0.3
```

Comment:
Need to bump it to 1.0.5. We already used a private 1.0.4 internally for a major code migration.